spec/README.md

```diff
@@ -58,8 +58,8 @@ Settings through two command line arguments, path and shell command.
 
 Tools with a rigid matching **can't evade** *[impedance
 mismatch](https://haacked.com/archive/2004/06/15/impedance-mismatch.aspx/)*.
-Resulting in an easy-to-use software, convenient in some scenarios but
-incapable in others.
+Resulting in software that is convenient and easy-to-use in a handful of scenarios,
+but sorely lacking in the vast majority of them.
```

Comment:
Do not express original intent. Also not Basic English.

Reply:
(reverted).  Merci !